python/libraries/bimvee/importIitYarp.py

```diff
@@ -240,23 +240,23 @@ def samplesToImu(inDict, **kwargs):
             tsOut[imuPtr] = ts # Just take the first ts for a group of samples
         sensorPrev = sensor
         if sensor == 0:
-            acc[imuPtr, 0] = value
+            acc[imuPtr, 1] = -value
```

Comment:
this should be + (x axis is index 1 but positive)

Reply:
sorry

Reply:
No - the sensor value from the raw data comes from line 242 (=0). the index in line 243 is the array that the value is being put into, where 0=x, 1=y, 2=z.